### ISSUE TITLE
Card component not rendered correctly

### ISSUE THREAD

**Issue by terjebra** [astrada/reason-react-toolbox#1]
Hi, 

First of all, great work on the Reason bindings for react-toolbox. I have just started to learn Reason/Reason-react and just found your bindings for react-toolbox, which I have previously used a lot with ReactJS.

I have been playing around with the [themeable example ](https://github.com/astrada/reason-react-toolbox-example-theme). I added the Card-component, but experienced some problems with the component compared to the [React Toolbox Card component]( http://react-toolbox.com/#/components/card).
I added the following
```reason
<ReactToolboxBundled.Card>
  <ReactToolboxBundled.CardTitle title = (ReasonReact.stringToElement "Card title")/>
</ReactToolboxBundled.Card>
```

which is outputted as 
```
<div data-react-toolbox="card" class="theme--card--8pay8nO5">
  <div data-react-toolbox="card" class="theme--card--8pay8nO5" title="title"></div>
</div>
 ```

If you look at the markup [here](http://react-toolbox.com/#/components/card), the output is:

```
<div data-react-toolbox="card" class="theme__card___I-1a7" >
  <div class="theme__cardTitle___WES7X theme__small___JPDNt">..</div>
</div>
```

The markup is clearly different in reason-react-toolbox. Do you know why is might be?  If you need anymore information, please let me know. Thanks.

**Comment by astrada** [comment]
OK, thanks for reporting this issue. The problem is that I don't correctly generate the external directive for submodules (e.g. https://github.com/astrada/reason-react-toolbox/blob/master/src/ReactToolboxBundled.re#L774).

**Comment by astrada** [comment]
I think I fixed this issue in version 0.2.2. Let me know if it still doesn't work.
